pkg/kv/kvclient/revision_reader.go

```diff
@@ -46,9 +46,16 @@ func GetAllRevisions(
 		return nil, pErr.GoError()
 	}
 
```

Comment:
@erikgrinaker any thoughts on the question below?

Reply:
Not likely. It would have to do additional key comparisons to figure out which file to read from next, and I suspect all of the cloning in this loop is going to dominate the cost anyway. But only benchmarks and profiles will say for sure.

Reply:
+1

Reply:
Sweet. I'll leave as is for now.